test.env.example

```diff
@@ -1,3 +1,7 @@
 BIGQUERY_TEST_ALT_DATABASE=
 BIGQUERY_TEST_NO_ACCESS_DATABASE=
 BIGQUERY_TEST_SERVICE_ACCOUNT_JSON='{}'
+
+DBT_TEST_USER_1="user:buildbot@dbtlabs.com"
+DBT_TEST_USER_2="user:buildbot@fishtownanalytics.com"
```

Comment:
```suggestion
DBT_TEST_USER_1="group:buildbot@dbtlabs.com"
DBT_TEST_USER_2="group:buildbot@fishtownanalytics.com"
```

Reply:
`buildbot@dbtlabs.com` + `buildbot@fishtownanalytics.com` are true aliases for one another. A grant to one is a grant to both. This causes tests to fail! We need to use genuinely different users/groups